www/js/iastate.js

```
@@ -1,361 +1,374 @@
- 		kkeys = [];
-	}
-	// clear non-konami-matching keypresses
-    else if ($.inArray(e.keyCode,[38,40,37,39,66,65]) == -1 ) {
-		kkeys = [];
-	}
-});
\ No newline at end of file
+$(document).ready(function () {
+  // IE console bug
+  // --------------------------------------------------
+  if (!window.console) console = { log: function () {} };
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)